website/docs/d/network_routed.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_network_routed"
+sidebar_current: "docs-vcd-data-source-network-routed"
+description: |-
+  Provides a vCloud Director Org VDC routed Network. This can be used to reference internal networks for vApps to connect.
+---
+
+# vcd\_network\_routed
+
+Provides a vCloud Director Org VDC routed Network. This can be used to reference internal networks for vApps to connect.
+
+Supported in provider *v2.5+*
+
+## Example Usage
+
+```hcl
+data "vcd_network_routed" "net" {
+  org  = "my-org" # Optional
+  vdc  = "my-vdc" # Optional
+  name = "my-net"
+}
+
+output "edge_gateway" {
+  value = data.vcd_network_routed.net.edge_gateway
+}
+
+output "gateway" {
+  value = data.vcd_network_routed.net.gateway
+}
+output "dhcp_start_address" {
+  value = tolist(data.vcd_network_routed.net.dhcp_pool)[0].start_address
+}
+
+output "dhcp_end_address" {
+  value = tolist(data.vcd_network_routed.net.dhcp_pool)[0].end_address
+}
+
+output "static_ip_start_address" {
+  value = tolist(data.vcd_network_routed.net.static_ip_pool)[0].start_address
+}
+
+output "static_ip_end_address" {
+  value = tolist(data.vcd_network_routed.net.static_ip_pool)[0].end_address
```

Comment:
For the record, it's quite a shame that there's no better way to access an element in a `TypeSet` directly - without `tolist`. Hopefully, Terraform will add such capability in the future and `TypeSet` will become a truly capable type.